### PR TITLE
Rust in Bitcoin / Rust in Ethereum

### PR DESCRIPTION
**Pull request by cryptoquick**
Breaks out Rust in Bitcoin and Rust in Ethereum sections. Adds some Rust in Bitcoin projects.

- [x] Rust Bitcoin
- [x] BDK
- [x] LDK
- [x] Sapio
- [x] LNP/BP
- [x] RGB
- [x] Internet2
- [x] Electrs
- [x] Rust Simplicity
- [x] Nakamoto
- [x] Bitmask
- [x] Nomic
- [x] Talaia

This is just a start, I know it probably won't work because there's a little plumbing to take of first. I can grant whoever wants to make contributions access to my PR. I'm not sure if the Ethereum work should happen in a separate branch. I can't really contribute to that. I'll also ask around to see if there's any projects I missed.

### CODE REVIEW

**Comment by cryptoquick** [comment]
@brson RFR

File: rib-bible.md [hunk 1]
```diff
@@ -525,14 +525,217 @@ GitHub:
...
+GitHub:
+
+- https://github.com/internet2-org/
+
+```toml
+repos = [
+"internet2-org/rust-aluvm",
+"internet2-org/rust-internet2",
+"internet2-org/rust-microservices",
+"internet2-org/stenc",
```
Comment (brson) [inline rib-bible.md:651]: These links appear to have moved to the internet2-WG. Can you update them?

File: template.md [hunk 2]
```diff
@@ -137,6 +133,52 @@ Bold entries here are especially relevant to blockchain projects.
...
+
+#### [LDK](https://github.com/lightningdevkit/rust-lightning)
+
+#### [Sapio](https://github.com/sapio-lang/sapio)
+
+#### [LNP/BP](https://github.com/LNP-BP)
+
+#### [RGB](https://github.com/rgb-org)
+
+#### [Internet2](https://github.com/internet2-org/)
```
Comment (brson) [inline template.md:152]: This link doesn't work. This org appears to be called internet2-WG now. Can you fix it?

**Comment by brson** [comment]
Thanks for doing all this. I like what you've done with the bitcoin and ethereum sections. I wonder if you wouldn't mind breaking up the bible the same way - put a bitcoin section and and ethereum section at the end.

I asked for two other changes just because the internet2-org links are broken.

https://github.com/rust-in-blockchain/ribbot will need to be updated to understand the new bitcoin/ethereum specific structure prior to the next newsletter, but I am willing to do that myself.

I am reading through all these projects now and will aim to put some kind of status update about progress in the rust bitcoin community in the next newsletter intro.

**Review by brson — CHANGES_REQUESTED** [review]

File: rib-bible.md [hunk 3]
```diff
@@ -525,14 +525,217 @@ GitHub:
...
+## RGB
+
+Media:
+
+- https://www.rgbfaq.com
+- https://rgbex.io
+
+GitHub:
+
+- https://github.com/rgb-org
```
Comment (brson) [inline rib-bible.md:630]: This link is 404. Can you fix it here and in the markdown template?

File: rib-bible.md (hunk 1 above)
Comment (cryptoquick) [reply rib-bible.md]: Updated

File: template.md (hunk 2 above)
Comment (cryptoquick) [reply template.md]: Updated

File: rib-bible.md (hunk 3 above)
Comment (cryptoquick) [reply rib-bible.md]: Updated

**Comment by cryptoquick** [comment]
> Thanks for doing all this. I like what you've done with the bitcoin and ethereum sections. I wonder if you wouldn't mind breaking up the bible the same way - put a bitcoin section and and ethereum section at the end.
> 
> I asked for two other changes just because the internet2-org links are broken.
> 
> https://github.com/rust-in-blockchain/ribbot will need to be updated to understand the new bitcoin/ethereum specific structure prior to the next newsletter, but I am willing to do that myself.
> 
> I am reading through all these projects now and will aim to put some kind of status update about progress in the rust bitcoin community in the next newsletter intro.

Thank you! And yeah, the links changed recently. Always gotta be on your toes with those wiley RGB people. Maxim and Olga are both very productive.

Thanks for taking on the ribbot work. Also, thanks for reading through, btw. I'd love to proofread your update for the next one.

**Pull request merged**